docs/README.md

```diff
@@ -749,6 +749,58 @@ This API enforces authorization. For example, you cannot modify a group you're n
 }
 ```
 
+#### Create Note And Add In Group
+
+**Description:** Create a note with or without blocks depeding of you request, and add it to the specify group.
+Must be logged with any role (user or admin).
+
+**Endpoint:** `POST /notes/create/:group_id`
```

Comment:
That's not very REST-like. You can just have a `POST /notes` and pass the `group_id` in the body.